doc/api/child_process.markdown

```diff
@@ -392,76 +392,202 @@ controlling terminal.
+the child's standard stream file descriptor. Indices 0, 1, and 2 correspond to
+the stdin, stdout, and stderr streams respectively. Indices 3 and up can be used
+to configure non-standard streams.
+
+The following are the possible values for the `options.stdio` configuration
+`Array`.
+
+* `Integer` file descriptor
+* `Stream` object
+* `'ignore'`
+* `'ipc'`
+* `'pipe'`
+* `null`
+* `undefined`
+
+For convenience, the `options.stdio` property also accepts one of the following
+`String` shorthands.
+
+| Shorthand   | Equivalent                       |
+|-------------|----------------------------------|
+| `'ignore'`  | `['ignore', 'ignore', 'ignore']` |
+| `'inherit'` | `[0, 1, 2]`                      |
+| `'pipe'`    | `['pipe', 'pipe', 'pipe']`       |
+
+When no value is provided, `options.stdio` defaults to `'pipe'`.
+
+##### `Integer` file descriptor
+
+Shares the parent process's open `Integer` file descriptor with a child process.
+The connection to the parent's file descriptor resource is duplicated and
+attached to a new file descriptor in the child. The new file descriptor
+corresponds to the index in the `options.stdio` configuration `Array`.
+
+The following example spawns a child process that shares the parent's stdin,
+stdout, and stderr `Integer` file descriptors.
 
     const spawn = require('child_process').spawn;
+    const child = spawn('node', ['--interactive'], { stdio: [0, 1, 2] });
+
+    child.on('close', function(code) {
+      process.exit(code);
+    });
+
+##### `Stream` object
+
+Shares a parent process's `Stream` object with a child process. These `Stream`
+objects must contain an open file descriptor. Therefore, valid objects are
+instances of the following classes.
+
+* `fs.ReadStream`
+* `fs.WriteStream`
+* `net.Socket`
+* `tty.ReadStream`
+* `tty.WriteStream`
+
+**NOTE:** Until the `'open'` event is emitted, `fs.ReadStream` and
+`fs.WriteStream` objects don't have an open file descriptor.
+
+The connection to the parent's `Stream` resource is duplicated and attached to a
+new file descriptor in the child. The new file descriptor corresponds to the
+index in the `options.stdio` configuration `Array`.
+
+The following example spawns a child process that shares the parent's stdin,
+stdout, and stderr `Stream` objects. The end result is identical to sharing the
+parent's standard stream file descriptors directly. See the `Integer` file
+descriptor example for more details.
+
+    const spawn = require('child_process').spawn;
+    const child = spawn('node', ['--interactive'], { 
+      stdio: [process.stdin, process.stdout, process.stderr]
+    });
+
+    child.on('close', function(code) {
+      process.exit(code);
+    });
+
+##### `'ignore'`
+
+Ignores the file descriptor in a child process. Ignoring indices 0, 1, or 2
+attaches the corresponding file descriptor to `/dev/null` in the child process.
+Ignoring indices 3 and up has no effect on the corresponding file descriptor.
```

Comment:
Its not so much it has no effect, but that its the default. Best to say: descriptors greater than the three standard ones are automaticcally closed by node when it spawns children to avoid bugs and security problems. If you have a descriptor open and want it to be opened in the child, you MUST use the .stdio property to do so. (or something like that).